src/ilcdlib/extension.py

```diff
@@ -30,6 +30,7 @@ class IlcdEpdExtension(OpenEpdExtension):
     dataset_version: str | None = None
     dataset_uuid: str | None = None
     production_location: str | None = None
+    epd_verifiers: list[Org] | None = None
```

Comment:
in extension it should be DTO with type field. Otherwise there is no way to understand which one is of what type.